### DISCUSSION TITLE
recreate superadmin with mysql

### DISCUSSION THREAD

**Discussion by gore89** [snipe/snipe-it#11861] [Q&A] [answered]
Hi,

I somehow deleted my superadmin. The LDAPS still works, bit the users dont get rights. I dont know how to assign super admin rights or admin rights for existing AD users. Could you help me, what i have to set in "permissions"?

kind regards

**Comment by snipe** [comment, marked as answer]
If you still have command line access, this should sort you out: https://snipe-it.readme.io/docs/create-an-admin

**Reply by gore89** [reply]
You are insane, i wasn't able to find that one..

**Reply by snipe** [reply]
I'm unclear if that's a compliment or not, but I wrote that script and the docs, so it was easy for me to find ;)
